api/v1/db/oracledb/game-dao.js

```diff
@@ -0,0 +1,61 @@
+const appRoot = require('app-root-path');
+const _ = require('lodash');
+
+
+const { serializeGames } = require('../../serializers/games-serializer');
+
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+const mergeRawGames = (rawGames) => {
+  const groupedRawGames = _.groupBy(rawGames, 'GAME_ID');
+  const mergedRawGames = _.map(groupedRawGames, (gameMetaDataArray) => {
+    if (gameMetaDataArray[0].CARD_NUMBER == null) {
+      gameMetaDataArray[0].tableCards = [];
+    } else {
+      gameMetaDataArray[0].tableCards = _.map(gameMetaDataArray, (data) => {
```

Comment:
You could use the ternary operator here instead of the `if..else` statement:
```js
gameMetaDataArray[0].tableCards = gameMetaDataArray[0].CARD_NUMBER === null ? [] : _.map(...)
```